docs/_mar-milestone_.md

```diff
@@ -0,0 +1,55 @@
+# Monthly Milestone - March 2020
+
+## Overview
+
+- Migration guide: Stop starting, start finishing!
+
+## Goals
+
+- [ ] Migration guide https://github.com/strongloop/loopback-next/issues/3935
+
+  _General runtime_
+
+  - [ ] [5] Which official LB3 components are not planned in LB4 #3956 (from Feb
+        milestone)
+  - [ ] [8] Migration Guide: Discuss life cycle differences between LB3 and LB4
+        #3935 (from Feb milestone)
+
+  _Authentication & Authorization_
+
+  - [ ] [8][migration/auth] Create a guide to describe how to migrate LB3 User
+        based authentication to LB4 #4521
+  - [ ] [8] How to migrate apps using loopback-component-passport #3958
+  - [ ] [5][migration/auth] Describe how to simulate ACLs with LB4 authorization
+        #4522
+
+- [ ] Miscellaneous
+  - [ ] Belongs-To relation tries to resolve for foreign key NULL #4372
+  - [ ] How to build models, repositories and controllers dynamically at runtime
+        #4296
+
```

Comment:
Let's add the story to update LB3-related docs to accommodate the transition to Maintenance LTS, see https://github.com/strongloop/loopback/issues/4306.